python/ml_croissant/ml_croissant/_src/operation_graph/operations/extract.py

```diff
@@ -18,9 +19,28 @@
 import pandas as pd
 
 
+def should_extract(encoding_format: str) -> bool:
```

Comment:
NIT: more than should_extract is seems is_supported_encoding or something similar?

Reply:
Acknowledged.

This function is used to know whether to extract or not a node. So I updated the docstring.